docs/sdk-and-tools/mxpy/mxpy-cli.md

```diff
@@ -110,17 +112,201 @@ One can alter the current configuration using the command `mxpy config set`. For
+
+```sh
+mxpy env set explorer_url
+```
+
+#### Setting the ask for confirmation flag
+
+If set to `true`, whenever sending a transaction, mxpy will display the transaction and will ask for your confirmation. To set the flag, use the following command:
+
+```sh
+mxpy env set ask_confirmation true
+```
+
+#### Dumping the active env config
+
+We can see the values set in our active env config. To do so, we use the following command:
+
+```sh
+mxpy env dump
+```
+
+#### Dumping all the available env configs
+
+We may have multiple env configs, maybe one for each network. To dump all the available env configs, we use the following command:
+
+```sh
+mxpy env list
+```
+
+#### Deleting a value from the active env config
+
+We can also delete the key-value pairs saved in the env config. For example, let's say we want to delete the explorer url, so we use the following command:
+
+```sh
+mxpy env delete explorer_url
+```
+
+#### Getting a value from the active env config
+
+If we want to see just the value of a specific env config key, we can use the following command:
+
+```sh
+mxpy env get <key-name>
+```
+
+#### Deleting an env config
+
+To delete an env config, we use the following command:
+
+```sh
+mxpy env remove <config-name>
+```
+
+#### Switching to a different env config
+
+To switch to a new env config, we use the following command:
+
+```sh
+mxpy env switch <config-name>
+```
+
+### Configuring wallets
+
+Wallets can be configured in the address config. From all the wallets configured, one must be set as the active wallet. This wallet will be the default wallet used through mxpy if not another wallet is provided via CLI args, such as `--pem`, `--keystore` or `--ledger`. Additionally, the `--sender` argument has been added and can be used to specify a certain sender from the address config (e.g. --sender alice).
+
+The values that are available for configuring wallets are the following:
+```json
+{
+    "kind": "",
+    "path": "",
+    "index": "",
+}
+```
+
+The `kind` field represents the wallet type and can be either `pem` or `keystore`. If the wallet is of type `keystore`, you'll be prompted to enter the wallet's password.
+
+The `path` field represents the absolute path to the wallet.
+
+The `index` field represents the index that will be used when deriving the wallet from the secret key. This field is optional, the default index is `0`.
+
+#### Creating a new address config
+
+To create a new address config, we use the following command:
+
+```sh
+mxpy address new <alias>
+```
+
+#### Setting the address config fields
+
+For a config to be valid, we need to set at least the `kind` and `path` fields. To do so, we use the following command:
+
+```sh
+mxpy address set kind pem
+mxpy address set path absolute/path/to/pem/wallet.pem
```

Comment:
Sorry I did not see this before - how does this work? Only for the default address?